atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTestTools.java

```diff
@@ -18,22 +18,32 @@
 import java.io.IOException;
 import java.net.InetSocketAddress;
 import java.net.Socket;
+import java.util.List;
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import org.joda.time.Duration;
 
 import com.jayway.awaitility.Awaitility;
 import com.jayway.awaitility.core.ConditionTimeoutException;
+import com.palantir.common.base.Throwables;
 
 /**
  * Utilities for ETE tests
  * Created by aloro on 12/04/2016.
  */
 public final class CassandraTestTools {
+    public static final int NUM_PARALLEL_TASKS = 32;
```

Comment:
This should be private (and can therefore be in the same block as the other `private static final` constant).